src/containers/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from "react";
+import SidebarAccordion from "./partials/SidebarAccordion";
+import Description from "./partials/Description";
+import { SidebarTitle } from "@dhi-gras/react-components";
+import { Paper } from "@material-ui/core";
+import LogoBlue from "./../../assets/DHI_Logo_Blue.png";
+
+import "./../../styles/scrollbar.css";
```

Comment:
this should be translated into `makeStyles` and shipped with the component, instead having it in a `.css` file

Reply:
These and the body css should be included as a css since it's a global customization of the `body` or the `scrollbar` which is not related to the classes from `makeStyles`. To get rid of the `.css` file I will add them to the `index.html`.